flask_restplus/fields.py

```diff
@@ -695,3 +699,63 @@ def clone(self, mask=None):
 
         data['mask'] = mask
         return Polymorph(mapping, **data)
+
+
+class Wildcard(List):
+    '''
+    Field for marshalling list of "unkown" fields.
+
+    :param cls_or_instance: The field type the list will contain.
+    '''
+    exclude = []
+    # keep a track of the last object
+    _idx = 0
+    # cache the flat object
+    _flat = None
+    _obj = None
+    _cache = []
+    _last = None
+
+    def _flatten(self, obj):
+        if obj == self._obj and self._flat:
+            return self._flat
+        if isinstance(obj, dict):
+            self._flat = obj.items()
+        else:
+            self._flat = []
+            attributes = inspect.getmembers(obj, lambda a: not(inspect.isroutine(a)))
+            for (key, val) in [a for a in attributes if not(a[0].startswith('__') and a[0].endswith('__'))]:
```

Comment:
I think this could be done in a single iteration:
Something like:
```python
def match_attribute(attr):  # defined once outside
    return not(inspect.isroutine(attr) or (attr.__name__.startswith('__') and attr.__name__.endswith('__')))

self._flat.extend(inspect.getmembers(obj, match_attributes))
```

Reply:
This is not completely doable, however, I managed to improve this a bit.